src/providers/factory.js

```diff
@@ -57,6 +57,14 @@ class ProviderFactory {
 
     return provider;
   }
+
+  // eslint-disable-next-line class-methods-use-this
```

Comment:
can use `static` modifier since we are not using `this`

Reply:
fixed